src/Components/ViewReport/Category/CategoryName.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import * as S from "../../styled/ViewReport/style";
+
+const CategoryName = (props) => {
```

Comment:
아 이건 맞춘건가..? 혜준이도 이렇게 하던데
아래처럼도 가능한거 알아두기!
```suggestion
const CategoryName = ({ SwName, frist, second, third, fourth }) => {
```